src/hpp/corbaserver/rbprm/rbprmbuilder.py

```diff
@@ -26,9 +26,12 @@
 #  trunk of the robot, and a set of robots describe the range of motion of each limb of the robot.
 class Builder(Robot):
     # # Constructor
-    def __init__(self, load=True):
+    def __init__(self, load=True, clientRbprm=None):
         self.tf_root = "base_link"
-        self.clientRbprm = RbprmClient()
+        if clientRbprm==None:
```

Comment:
`None` is a  singleton in Python, therefore you should use `if clientRbprm is None:`

Reply:
Fix on syntax done in the last commit.